dockerfiles/e2e/Dockerfile

```diff
@@ -5,11 +5,16 @@ ENV DISPLAY=':20'
 USER root
 
 RUN apt-get update && apt-get install && \
+    apt-get install -y ftp && \
     apt-get install -y nodejs && \
     apt-get install -y npm && \
     npm install -g typescript && \
     apt-get install x11vnc -y
 
+RUN mkdir /tmp/e2e && \
+    chmod -R o+t /tmp/e2e && \
+    chmod -R 777 /tmp/e2e
```

Comment:
This place not understandable for me, why are you set twice different permissions to the same folder?

Reply:
I'm not sure how can I set permission and a sticky bit in one command?

Reply:
But the second command overrides the first
- the first command permissions   ```drwxrwxr-t ```
- the second command permissions ```drwxrwxrwx```

Reply:
Heh, that's true. Good point. So I'll remove the first one.